components/compose/awesomebar/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Description for the button to accept the search suggestion and continue editing the search. -->
+    <string name="mozac_browser_awesomebar_edit_suggestion">Accept and edit suggestion</string>
```

Comment:
I think you forgot to use this for the content description.

Reply:
Oops, yeah, I refactored this multiple times.